### DISCUSSION TITLE
Fuji PXF4 question

### DISCUSSION THREAD

**Discussion by Dgocof** [artisan-roaster-scope/artisan#1098] [General]
Does anyone have a suggestion as to why my SV slider would be different than the SV on the Fuji display. As you can see the further I go up the more the value is off. I can post screen shots if you need to see my setup. Temperatures are more or less the Fuji display temp rather than my SV. Therefore when I try to follow a back round BT the SV will follow the BT but the actual temperature  is looking like it is the lower setpoint on the Fuji rather than the back round BT
When my
SV slider is set to:         ----          My Fuji SV says:
       267          -  -----------------                 224
       419            - -----------------                   333
     500           -  --------------- -                            391

**Comment by MNS19** [comment]
Hi there, did you ever figure this out? I’m encountering a very similar issue with my Fuji PXG4 PID and Artisan.

The Fuji displays roughly 50% of the slider SV.
For example
SV slider is set to: ---- My Fuji SV says:
100 - ----------------- 50
212 - ----------------- 104.7
302 - --------------- - 149.2
The only time artisan and the PXG are in sync is when I set SV slider at 0. 

Thanks!

**Comment by Dgocof** [comment]
Sorry but there were so many settings that I tried before it worked. It was a while back but I did find one eventually . Channel 1 PID (if you are using PID 1)   Try 60 to 999 from 100. SV upper limit. I'm not sure.

**Reply by MNS19** [reply]
Great, thank you for the reply, much appreciated! 

Just to clarify: what do you mean by "Try 60 to 999 from 100" ?
- do you mean try inputting anything from 60-999 into the "SV upper limit" field in artisan?


Thank you!

**Comment by Dgocof** [comment]
In the fuji manual, on the left hand side each setting has a number in the
parameter column. Page 8. Line 60 SVH upper limit Ch1 PID control
parameters.
https://www.instrumart.com/assets/PXG4_Short_Form_Manual.pdf

On Thu, Apr 18, 2024 at 2:48 PM MNS19 ***@***.***> wrote:

> Great, thank you for the reply, much appreciated!
>
> Just to clarify: what do you mean by "Try 60 to 999 from 100" ?
>
>    - do you mean try inputting anything from 60-999 into the "SV upper
>    limit" field in artisan?
>
> Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/1098#discussioncomment-9159537>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A36HWNYOHPDZA3MTS4RYOTLY6AWQ5AVCNFSM6AAAAAAUYIZENOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCNJZGUZTO>
> .
> You are receiving this because you authored the thread.Message ID:
> <artisan-roaster-scope/artisan/repo-discussions/1098/comments/9159537@
> github.com>
>


-- 
Durango Coffee Company
DurangoCoffee.com
970-385-0276

**Reply by MNS19** [reply]
Excellent, thank you very much! I'll give it a go 👍

**Comment by Dgocof** [comment]
Let me know if that worked.
Good luck

On Thu, Apr 18, 2024 at 3:29 PM MNS19 ***@***.***> wrote:

> Excellent, thank you very much! I'll give it a go 👍
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/1098#discussioncomment-9159812>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A36HWN2PEWBSCQMFM7LXCATY6A3NLAVCNFSM6AAAAAAUYIZENOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCNJZHAYTE>
> .
> You are receiving this because you authored the thread.Message ID:
> <artisan-roaster-scope/artisan/repo-discussions/1098/comments/9159812@
> github.com>
>


-- 
Durango Coffee Company
DurangoCoffee.com
970-385-0276

**Reply by MNS19** [reply]
Absolutely, will do

**Comment by Dgocof** [comment]
[TN5A2227b-E_tfjretsyli.pdf](https://github.com/artisan-roaster-scope/artisan/files/15042831/TN5A2227b-E_tfjretsyli.pdf)
Sorry try page 41 for #60 in this document.

**Comment by MNS19** [comment]
Ah! Excellent, thanks for clarifying! really appreciate that.
Im going to try it out tonight. Just so that I am clear - do you remember if you input the resulting decimal number (32) into artisan under which register box below?

![ports config artisan](https://github.com/artisan-roaster-scope/artisan/assets/102933824/34f473e3-fbd5-4c4b-8cd1-68896946596d)

**Comment by Dgocof** [comment]
Not sure if we are talking the same thing. The number 60 corresponds to the
Fuji setup not Artisan. The first document I sent shows you how to access
the setup menu on the Fuji.
My register settings are different from yours in Artisan. I have attached
mine for you to see.

On Fri, Apr 19, 2024 at 1:40 PM MNS19 ***@***.***> wrote:

> Ah! Excellent, thanks for clarifying! really appreciate that.
> Im going to try it out tonight. Just so that I am clear - do you remember
> if you input the resulting decimal number (32) into artisan under which
> register box below?
>
> ports.config.artisan.jpg (view on web)
> <https://github.com/artisan-roaster-scope/artisan/assets/102933824/34f473e3-fbd5-4c4b-8cd1-68896946596d>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/1098#discussioncomment-9170378>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A36HWN2NFXQT5IN4QAVUGMDY6FXKLAVCNFSM6AAAAAAUYIZENOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCNZQGM3TQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <artisan-roaster-scope/artisan/repo-discussions/1098/comments/9170378@
> github.com>
>


-- 
Durango Coffee Company
DurangoCoffee.com
970-385-0276

**Reply by MNS19** [reply]
Ah yes, I see what you're saying.
When i access that menu on the Fuji my Svh is 499 (ie maximum) but changing that value (even to 0) doesn't seem to affect the SV display at all.
Thanks for the screenshot - maybe it didn't attach? Curious to take a look!

**Comment by Dgocof** [comment]
[image: image.png]

I think I set my SVH to 999.

On Fri, Apr 19, 2024 at 3:55 PM MNS19 ***@***.***> wrote:

> Ah yes, I see what you're saying.
> When i access that menu on the Fuji my SVH is 499 (ie maximum) but
> changing that value (even to 0) doesn't seem to affect the SV display at
> all.
> Thanks for the screenshot - maybe it didn't attach? Curious to take a look!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/1098#discussioncomment-9171237>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A36HWN26UXI5XE7WYEG32CDY6GHF3AVCNFSM6AAAAAAUYIZENOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCNZRGIZTO>
> .
> You are receiving this because you authored the thread.Message ID:
> <artisan-roaster-scope/artisan/repo-discussions/1098/comments/9171237@
> github.com>
>


-- 
Durango Coffee Company
DurangoCoffee.com
970-385-0276

**Comment by Dgocof** [comment]
[image: image.png]
Another thing maybe to check is your duty % in control (Artisan)

On Fri, Apr 19, 2024 at 4:10 PM Durango Coffee Company <
***@***.***> wrote:

> [image: image.png]
>
> I think I set my SVH to 999.
>
> On Fri, Apr 19, 2024 at 3:55 PM MNS19 ***@***.***> wrote:
>
>> Ah yes, I see what you're saying.
>> When i access that menu on the Fuji my SVH is 499 (ie maximum) but
>> changing that value (even to 0) doesn't seem to affect the SV display at
>> all.
>> Thanks for the screenshot - maybe it didn't attach? Curious to take a
>> look!
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/artisan-roaster-scope/artisan/discussions/1098#discussioncomment-9171237>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A36HWN26UXI5XE7WYEG32CDY6GHF3AVCNFSM6AAAAAAUYIZENOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCNZRGIZTO>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> <artisan-roaster-scope/artisan/repo-discussions/1098/comments/9171237@
>> github.com>
>>
>
>
> --
> Durango Coffee Company
> DurangoCoffee.com
> 970-385-0276
>


-- 
Durango Coffee Company
DurangoCoffee.com
970-385-0276

**Reply by MNS19** [reply]
I'll double check the duty also, cheers! 
Screenshots/photos still not coming through for some reason 🤷🏻‍♂️

**Reply by Terracotta-6** [reply]
![ad8d0d44513bc1c1db843689eb59e4c](https://github.com/artisan-roaster-scope/artisan/assets/149906183/35f0fc73-7e7c-4978-84e5-c141335436a6)

**Reply by Terracotta-6** [reply]
Decode ulnt16

**Reply by Terracotta-6** [reply]
https://www.bigloose.com/v11/index.php/home/archive/artisan-roasting-with-the-pid-fuji-pxf-and-test-of-maxwell-mta

**Reply by MNS19** [reply]
Thanks! Yes, Nico's blog is the exact tutorial I have been following. He also very kindly helped me out via email and everything is working as per his instructions *except* for this final piece of the puzzle. I do notice when I choose the PID button (rather than meter: modbus) under device assignment then the Fuji displays the exact same SV as the Artisan slider and all seems well, BUT it follows the background curve of ET rather than BT (the point of Nico's tutorial) It could just be me missing something very basic of course.

**Comment by Dgocof** [comment]
sint 16 for me. I'll try to post them through git hub.

On Fri, Apr 19, 2024 at 5:13 PM Terracotta-6 ***@***.***>
wrote:

> Decode ulnt16
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/1098#discussioncomment-9171505>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A36HWN4FPUZD4JPXXOQFIILY6GQI3AVCNFSM6AAAAAAUYIZENOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCNZRGUYDK>
> .
> You are receiving this because you authored the thread.Message ID:
> <artisan-roaster-scope/artisan/repo-discussions/1098/comments/9171505@
> github.com>
>


-- 
Durango Coffee Company
DurangoCoffee.com
970-385-0276

**Comment by Dgocof** [comment]
<img width="568" alt="image-2" src="https://github.com/artisan-roaster-scope/artisan/assets/117209911/f423b25c-755c-4738-a682-28bb8d9553f8">
<img width="1296" alt="image" src="https://github.com/artisan-roaster-scope/artisan/assets/117209911/0baf54c3-3c11-48a6-9aa8-4656e0c50dc1">

**Reply by MAKOMO** [reply]
Which Artisan version are you using here? The MODBUS PID dialog should look different now then in your screenshot.

To me it seems that your SV register number is wrong. According to the PXF modbus register map manual SV1 relative address is 00F0 in hex which translates to 240. To this, as noted in the same manual, there is an offset of 2000 to be added resulting in the 2240.

**Reply by MNS19** [reply]
Hi Marko , thank you for chiming in on this. I believe the settings in the screenshot above are working for the user (@dgocof) who's been kindly helping me troubleshoot on this thread. 
Do you know if the same offset of 2000 is required for the Fuji PXG4? I can't seem to find that information in the PXG4 manual, nor can I find a Modbus Register Map specific to the PXG4 - I assume it is the same as the PXF? Thank you!

**Comment by Dgocof** [comment]
Yes I followed most of those settings except the D in the PID were incorrect on big loose.

**Reply by Terracotta-6** [reply]
artisan san modbus & Parameter loader softwarerameter loader software makes it easy to verify that register values are correct

**Reply by Terracotta-6** [reply]
https://www.fujielectric.com/products/sensors_measurements/instruments/product_detail/products_controller_PXF.html

**Reply by MNS19** [reply]
> artisan san modbus & Parameter loader softwarerameter loader software makes it easy to verify that register values are correct

Thank you for chiming in. Appreciate it

**Comment by MNS19** [comment]
Thanks all, it looks to be working now! The PXG4 requires an offset of 1000. So instead of 240 register 1240 works. Thanks @MAKOMO for pointing that possibility out & thanks @Dgocof   for your assistance. Much appreciated. I'll be testing out properly tomorrow and will confirm but as of now the Fuji SV is in sync with Artisans SV slider and following the BT background curve nicely. Cheers all!

**Reply by MAKOMO** [reply]
Well, the title of this discussion talks about PXF....

**Reply by MNS19** [reply]
That's true, thanks for pointing it out - looks like misread the title initially when i posted a reply. I thought it said pxg.
